infra/core/key-vault/platform-key-vault.parameters.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
as this file is **new**, the standard is bicepParameters for this one

Reply:
Doh!  - I'll update it.

Reply:
@DaRoszko - I've converted it to bicep parameters and here's the successful build:

https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=406125&view=logs&j=6263ae07-080d-5a18-f654-fdb20c6ea0e3&t=6263ae07-080d-5a18-f654-fdb20c6ea0e3